play-nunjucks-play-30/package.json

```diff
@@ -4,7 +4,9 @@
     "audit": "better-npm-audit audit"
   },
   "dependencies": {
-    "nunjucks": "^3.2.4",
-    "better-npm-audit": "^3.7.3"
+    "nunjucks": "3.2.4"
+  },
+  "devDependencies": {
+    "better-npm-audit": "3.7.3"
```

Comment:
with it in dependencies, and the way were installing, better-npm-audit would possibly be getting added to the artefact